target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -5,44 +5,45 @@ RAMFS_COPY_BIN='fw_printenv fw_setenv'
 RAMFS_COPY_DATA='/etc/fw_env.config /var/lock/fw_printenv.lock'
 
 platform_check_image() {
-	return 0;
+        return 0;
 }
 
 platform_do_upgrade() {
-	case "$(board_name)" in
-	buffalo,wxr-2533dhp)
-		buffalo_upgrade_prepare_ubi
-		CI_ROOTPART="ubi_rootfs"
-		nand_do_upgrade "$1"
-		;;
-	compex,wpq864|\
-	netgear,d7800 |\
-	netgear,r7500 |\
-	netgear,r7500v2 |\
-	netgear,r7800 |\
-	qcom,ipq8064-ap148 |\
-	qcom,ipq8064-ap161)
-		nand_do_upgrade "$1"
-		;;
-	zyxel,nbg6817)
-		zyxel_do_upgrade "$1"
-		;;
-	linksys,ea7500-v1 |\
-	linksys,ea8500)
-		platform_do_upgrade_linksys "$1"
-		;;
-	tplink,c2600)
-		PART_NAME="os-image:rootfs"
-		MTD_CONFIG_ARGS="-s 0x200000"
-		default_do_upgrade "$1"
-		;;
-	tplink,vr2600v)
-		PART_NAME="kernel:rootfs"
-		MTD_CONFIG_ARGS="-s 0x200000"
-		default_do_upgrade "$1"
-		;;
-	*)
-		default_do_upgrade "$1"
-		;;
-	esac
+        case "$(board_name)" in
+        askey,rt4230w-rev6 |\
+        compex,wpq864 |\
+        netgear,d7800 |\
+        netgear,r7500 |\
+        netgear,r7500v2 |\
+        netgear,r7800 |\
+        qcom,ipq8064-ap148 |\
+        qcom,ipq8064-ap161)
+                nand_do_upgrade "$1"
+                ;;
+        buffalo,wxr-2533dhp)
+                buffalo_upgrade_prepare_ubi
+                CI_ROOTPART="ubi_rootfs"
+                nand_do_upgrade "$1"
+                ;;
+        zyxel,nbg6817)
+                zyxel_do_upgrade "$1"
+                ;;
+        linksys,ea7500-v1 |\
+        linksys,ea8500)
+                platform_do_upgrade_linksys "$1"
+                ;;
+        tplink,c2600)
+                PART_NAME="os-image:rootfs"
+                MTD_CONFIG_ARGS="-s 0x200000"
+                default_do_upgrade "$1"
+                ;;
+        tplink,vr2600v)
+                PART_NAME="kernel:rootfs"
+                MTD_CONFIG_ARGS="-s 0x200000"
+                default_do_upgrade "$1"
+                ;;
+        *)
+                default_do_upgrade "$1"
+                ;;
+        esac
```

Comment:
Please don't change indent here.

Reply:
I don't even know how that happened but it's fixed now